lib/cpp/test/TServerSocketTest.cpp

```diff
@@ -46,7 +40,8 @@ BOOST_AUTO_TEST_CASE( test_bind_to_address )
     accepted->close();
     sock1.close();
 
-    TServerSocket sock2("this.is.truly.an.unrecognizable.address.", m_serverPort);
+    std::cout << "An error message from getaddrinfo on the console is expected:" << std::endl;
+    TServerSocket sock2("257.258.259.260", m_serverPort);
```

Comment:
Initially I used a bogus DNS name however it took 15 seconds to fail.  This was much faster. :>